.github/workflows/master_lambda.yaml

```diff
@@ -12,6 +12,8 @@ env:
   AWS_DEFAULT_REGION: us-east-1
   GOOGLE_CALENDAR_API_KEY: ${{ secrets.GOOGLE_CALENDAR_API_KEY }}
   GITHUB_PERSONAL_ACCESS_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+  TWITTER_BEARER_TOKEN: ${{ secrets.TWITTER_CONSUMER_KEY }}
```

Comment:
Should probably call this `TWITTER_CONSUMER_KEY` right?

EDIT: yea we definitely need this to match https://github.com/dvargas92495/roam-js-extensions/pull/66/files#diff-5d4f96efb0abeb3fa4cc55154d4f9307R47

Reply:
My bad, fixed!